<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfig</class>
 <widget class="QWidget" name="GeneralConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>382</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="1">
    <widget class="QCheckBox" name="alwaysUseDialog">
     <property name="text">
      <string>Always list the applications in a menu</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="showMaximize">
     <property name="text">
      <string>Show Maximize Button</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="showMinimize">
     <property name="text">
      <string>Show Minimize Button</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="showTitle">
     <property name="text">
      <string>Show Window Title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="showClose">
     <property name="text">
      <string>Show Close Button</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="showIcon">
     <property name="text">
      <string>Show Icon</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="windowDrag">
     <property name="text">
      <string>Enable window drag</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="borderlessMaximizedWindow">
     <property name="text">
      <string>Don't show window title when maximized</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Appearance:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Window management:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
